frontend/device/sdl/device.lua

```diff
@@ -123,7 +124,7 @@ local Emulator = Device:new{
 local UbuntuTouch = Device:new{
     model = "UbuntuTouch",
     hasFrontlight = yes,
-    home_dir = nil,
+    home_dir = nil, --  ~/Documents ?
```

Comment:
Just the default for SDL should do afaik. I.e., `os.getenv("XDG_DOCUMENTS_DIR")` which will usually lead to `$HOME/Documents`.

```suggestion
```

Reply:
the SDL default for now is `os.getenv("HOME")` only appimage is `os.getenv("XDG_DOCUMENTS_DIR")` should i have UT be that or change the sdl default?

Reply:
I see no reason to make a distinction, so I'd change the default.

Reply:
done